styles/scss/globals.scss

```diff
@@ -20,6 +24,97 @@
   }
 }
 
+//Dark theme
+.dark body{
+  background-color: $textPrimaryColor;
+  color: $secondaryColor;
+  
+  #leftbar,
+  #settingLeft,
+  #task_sidebar,
+  #task_sidebar_list {
+    background-color: $textSecondaryColor;
+    color: $secondaryColor;
+
+    h4, a, span {
+      color: $secondaryColor;
+    }
+  }
+
+  
+  
+  #taskEditor,
+  #menuBox,
+  #settingRight,
+  #addProject,
+  #newTask,
+  #noticeContainer,
+  #task_header,
+  #task_main_container,
+  #project_popup_window,
+  #add_label {
+    background-color: $textPrimaryColor;
+    color: $secondaryColor;
+    border: $textSecondaryColor 0.2px solid;
+
+    a, span {
+      color: $secondaryColor;
+    }
+  }
+
+  #listBox,
+  #viewHeader,
+  #content_holder {
+    background-color: $textPrimaryColor;
+    color: $secondaryColor;
+  }
+
+  #icon {
+    color: $secondaryColor;
+  }
+
+  #settingRight {
+    div {
+      border-top: 1px solid $textPrimaryColor;
+    }
+  }
+
+  #task_tag_item,
+  #add_btn {
+    background-color: $textPrimaryColor;
+    &:hover {
+      background-color: rgba($color: $heaveGrayColor, $alpha: 0.8);
+    }
+  }
+
+  #task_sidebar_button,
+  #action_menu_btn {
+    background-color: none;
+    color: $secondaryColor;
+    &:hover {
+      background-color: rgba($color: $heaveGrayColor, $alpha: 0.8);
+    }
+  }
+
+  #leftbar_item:hover,
+  #project_item:hover,
+  #option_link:hover,
+  #setting_item:hover {
+    background-color: rgba($color: $heaveGrayColor, $alpha: 0.8);
+  }
+
+  #project_item,
+  #right_menu {
+    background-color: none;
+    color: $secondaryColor;
+  }
+
+  #task_discription,
+  #task_label {
+    color: $grayColor;
+  }
+}
+
```

Comment:
I think I understand why you chose to use IDs. If using classNames doesn't achieve the desired effect, please disregard my previous comments about IDs. However, if it's also feasible with classNames, I suggest trying to maintain consistency for uniformity.

Reply:
I tried to use className but it didn't work, so maybe just keep it this way.